tests/tensorflow_autolog/test_tensorflow2_autolog.py

```diff
@@ -925,9 +925,9 @@ def test_import_tf_keras_with_fluent_autolog_enables_tf_autologging():
 
     assert not autologging_is_disabled(mlflow.tensorflow.FLAVOR_NAME)
 
-    # NB: For backwards compatibility, fluent autologging enables TensorFlow and
-    # Keras autologging upon tf.keras import in TensorFlow 2.5.1
-    if Version(tf.__version__) != Version("2.5.1"):
+    # NB: In Tensorflow >= 2.6, we redirect keras autologging to tensorflow autologging
+    # so the original keras autologging is disabled
+    if Version(tf.__version__) >= Version("2.6"):
         assert autologging_is_disabled(mlflow.keras.FLAVOR_NAME)
```

Comment:
In `tensorflow < 2.6`, does `import tensorflow.keras` enable keras autologging?

Reply:
No.

Reply:
Good catch. We haven't import keras in the test, add `import keras` to make testing meaningful.

Reply:
In tensorflow < 2.6, keras is not a dependency of tensorflow, in the cross test we cannot import keras module